tests/usecase.py

```diff
@@ -6,19 +6,25 @@
 
 import subprocess
 import time
+import os
 
+# Directory for cloning Apache Kafka sources to.
+# It will only be used if 'version' is not specified.
+# Otherwise Apache Kafka binary distrubution will be downloaded to
+# a cluster data dir ( trivup/tmp/TestCluster/KafkaBrokerApp/kafka/conf['version'] )
+kafka_dir=os.path.join(os.path.expanduser('~'), "src", "kafka")
 
 if __name__ == '__main__':
     cluster = Cluster('TestCluster', 'tmp')
 
     # One ZK
-    zk1 = ZookeeperApp(cluster, bin_path='/home/maglun/src/kafka/bin/zookeeper-server-start.sh')
+    zk1 = ZookeeperApp(cluster, bin_path=os.path.join(kafka_dir, '/bin/zookeeper-server-start.sh'))
 
-    # Two brokers
-    conf = {'replication_factor': 3, 'num_partitions': 4}
-    broker1 = KafkaBrokerApp(cluster, conf, kafka_path='/home/maglun/src/kafka')
-    broker2 = KafkaBrokerApp(cluster, conf, kafka_path='/home/maglun/src/kafka')
-    broker3 = KafkaBrokerApp(cluster, conf, kafka_path='/home/maglun/src/kafka')
+    # Multiple Kafka brokers
+    conf = {'version': '2.2.0', 'replication_factor': 3, 'num_partitions': 4, 'kafka_path': kafka_dir}
```

Comment:
With version specified (binary download) we dont need kafka_path (source tree), right?